### ISSUE TITLE
Unable to render the Category controller as one of the attribute is an object

### ISSUE THREAD

**Issue by RapotOR** [Herzult/HerzultForumBundle#32]
The problem appears since this commit in the master branch (2.1-dev) : 

https://github.com/symfony/symfony/commit/789d5ad20f5d3e63265b361c8c16230c2c070a96#src/Symfony/Bundle/FrameworkBundle/HttpKernel.php

In `Category\show.html.twig`

```
 {% render 'HerzultForumBundle:Topic:list' with {'category': category, ... %}
```

Regards,

**Comment by Herzult** [comment]
Fixed in 816823ddd317ba1884ce7bc02082ce99fe0dc8ba

**Comment by RapotOR** [comment]
thanks :)
